docs/sources/mobile-app/push-notifications/index.md

```diff
@@ -82,4 +83,31 @@ You can also enable or disable Do Not Disturb override for **important** notific
 
 On-call shift notifications are sent to announce upcoming shifts, roughly ~15 minutes in advance.
 
-To enable or disable on-call shift notifications, use the **On-call shift notifications** section in the **Push notifications** settings.
+To enable or disable on-call shift notifications, use the **info notifications** section in the **Push notifications** settings.
+
+### Shift swap notifications
+
+Shift swap notifications are generated when a [shift swap ][shift-swaps] is created,
+informing all users in the on-call schedule (except the initiator) about potential schedule changes.
+
+Shift swap notifications are sent 4 weeks ahead of the requested shift swap, or shortly after creation in case
+the shift swap start time is less than 4 weeks away, but always during users' working hours (by default 9am-5pm on
+weekdays, according to the user's mobile device timezone).
+
+As long as the shift swap request is open, there will be follow-up push notifications at the following intervals before the swap start:
```

Comment:
this feels like a bit too much detail for this page. maybe we can just link to the shift swaps page where we also explain this whole timeline.